lib/main.coffee

```diff
@@ -91,6 +97,23 @@ Notifications =
       Notifications.notificationsPanelView = new NotificationsPanelView
       Notifications.notificationsPanel = atom.workspace.addBottomPanel(item: Notifications.notificationsPanelView.getElement())
 
+  createLog: (serialized) ->
```

Comment:
I copied that out of the [Flight Manual](https://flight-manual.atom.io/hacking-atom/sections/package-active-editor-info/#summary) since this is my first time dealing with docks.